pkg/scheduler/factory.go

```diff
@@ -113,181 +109,104 @@ type Config struct {
+	PVCLister corelisters.PersistentVolumeClaimLister
 	// a means to list all services
-	serviceLister corelisters.ServiceLister
+	ServiceLister corelisters.ServiceLister
 	// a means to list all controllers
-	controllerLister corelisters.ReplicationControllerLister
+	ControllerLister corelisters.ReplicationControllerLister
 	// a means to list all replicasets
-	replicaSetLister appslisters.ReplicaSetLister
+	ReplicaSetLister appslisters.ReplicaSetLister
 	// a means to list all statefulsets
-	statefulSetLister appslisters.StatefulSetLister
+	StatefulSetLister appslisters.StatefulSetLister
 	// a means to list all PodDisruptionBudgets
-	pdbLister policylisters.PodDisruptionBudgetLister
+	PdbLister policylisters.PodDisruptionBudgetLister
 	// a means to list all StorageClasses
-	storageClassLister storagelistersv1.StorageClassLister
+	StorageClassLister storagelistersv1.StorageClassLister
 	// a means to list all CSINodes
-	csiNodeLister storagelistersv1beta1.CSINodeLister
+	CSINodeLister storagelistersv1beta1.CSINodeLister
 	// a means to list all Nodes
-	nodeLister corelisters.NodeLister
+	NodeLister corelisters.NodeLister
 	// a means to list all Pods
-	podLister corelisters.PodLister
+	PodLister corelisters.PodLister
 
 	// Close this to stop all reflectors
 	StopEverything <-chan struct{}
 
-	schedulerCache internalcache.Cache
+	SchedulerCache internalcache.Cache
 
 	// RequiredDuringScheduling affinity is not symmetric, but there is an implicit PreferredDuringScheduling affinity rule
 	// corresponding to every RequiredDuringScheduling affinity rule.
 	// HardPodAffinitySymmetricWeight represents the weight of implicit PreferredDuringScheduling affinity rule, in the range 0-100.
-	hardPodAffinitySymmetricWeight int32
+	HardPodAffinitySymmetricWeight int32
 
 	// Handles volume binding decisions
-	volumeBinder *volumebinder.VolumeBinder
+	VolumeBinder *volumebinder.VolumeBinder
 
 	// Always check all predicates even if the middle of one predicate fails.
 	alwaysCheckAllPredicates bool
 
 	// Disable pod preemption or not.
-	disablePreemption bool
+	DisablePreemption bool
 
 	// percentageOfNodesToScore specifies percentage of all nodes to score in each scheduling cycle.
-	percentageOfNodesToScore int32
+	PercentageOfNodesToScore int32
 
-	bindTimeoutSeconds int64
+	BindTimeoutSeconds int64
 
-	podInitialBackoffSeconds int64
+	PodInitialBackoffSeconds int64
 
-	podMaxBackoffSeconds int64
+	PodMaxBackoffSeconds int64
 
 	enableNonPreempting bool
 
 	// framework configuration arguments.
-	registry                     framework.Registry
-	plugins                      *schedulerapi.Plugins
-	pluginConfig                 []schedulerapi.PluginConfig
-	pluginConfigProducerRegistry *plugins.ConfigProducerRegistry
+	Registry                     framework.Registry
+	Plugins                      *schedulerapi.Plugins
+	PluginConfig                 []schedulerapi.PluginConfig
+	PluginConfigProducerRegistry *plugins.ConfigProducerRegistry
```

Comment:
add a space here and group together the unexported fields.

Reply:
do you mean to group all the unexported fields together or just the ones after `PluginConfigProducerRegistry`?

Reply:
all the unexported fields.